site/src/pages/components/selectmenu.mdx

```diff
@@ -0,0 +1,257 @@
+---
+menu: Proposals
+name: Selectmenu Element (Explainer)
+layout: ../../layouts/ComponentLayout.astro
+---
+
+import SelectAnatomy from '../../components/select-anatomy'
+import Image from '../../components/image'
```

Comment:
Don't forget the `astro` extension

import Image from '../../components/image.astro'

Reply:
done